### DISCUSSION TITLE
cannot port working code to wroover

### DISCUSSION THREAD

**Discussion by cemox** [schreibfaul1/ESP32-audioI2S#632] [General]
When I try to port the same code which works on ESP32 to ESP32-WROVER, it does not work.
 It is a simple streaming online radio sketch using VS-1053B. Not using PSRAM for now. 

```
// Connections for ESP32
#define VS1053_MISO     19
#define VS1053_MOSI     23
#define VS1053_SCK      18
#define VS1053_XCS      5   
#define VS1053_DCS      16 
#define VS1053_DREQ     4  
```
```
// Connections for ESP32-WROVER
#define VS1053_MISO     19
#define VS1053_MOSI     23
#define VS1053_SCK      18
#define VS1053_XCS      5   
#define VS1053_DCS      25 
#define VS1053_DREQ     26  
```
The only difference is DCS and DREQ pins because GPIO16 is not available for WROVER. 

In the terminal output I see that connection is OK and it is processing the stream data. But, there is no sound output. Any help will be appreciated. 

```
Homepage:     https://www.acikradyo.com.tr
lastURL:      http://stream.34bit.net/ar.mp3
DEBUG:        stream ready
DEBUG:        StreamTitle=''
STREAMTITLE:  
title  size 0 

DEBUG:        slow stream, dropouts are possible
DEBUG:        slow stream, dropouts are possible
DEBUG:        slow stream, dropouts are possible
DEBUG:        slow stream, dropouts are possible
DEBUG:        slow stream, dropouts are possible
DEBUG:        slow stream, dropouts are possible
DEBUG:        slow stream, dropouts are possible 
```

**Comment by schreibfaul1** [comment]
Hello, this is a question about the vs1053_ext library. I don't want to develop it further because of the old HW and have archived it. An external DAC such as the PCM5102A is a good alternative. Please understand that I am closing this topic here.

**Comment by cemox** [comment]
I am terribly sorry I forgot that.

Cheers,
Cem

Wolle ***@***.***>, 22 Ara 2023 Cum, 11:12 tarihinde şunu
yazdı:

**Comment by cemox** [comment]
Do you have a "ko-fi com" account or something like that? (patreon etc.).
This is a wonderful library and obviously you spared lots of time and
energy. I would like to order a cup of coffe or beer, or make a small
donation to show my gratitude :)

Best regards,
cem

Cem Berik ***@***.***>, 22 Ara 2023 Cum, 11:37 tarihinde şunu yazdı:
